src/main/resources/templates/stats.html

```diff
@@ -23,12 +24,15 @@
             <div class="border border-dark thank-you">
                 <p class="h2 mt-2 text-center">Config</p>
                 <p>
-                    Hints enabled: <span th:text="${hintsEnabled}"></span> <br/>
+                    Hints enabled: <span th:text="${hintsEnabled}"></span><br/>
                     Reason enabled: <span th:text="${reasonEnabled}"></span><br/>
                     CTF-mode enabled: <span th:text="${ctfModeEnabled}"></span><br/>
-                    Springdoc enabled: <span th:text="${springdockenabled}"></span><br/>
-                    Swagger enabled: <span th:text="${swaggerEnabled}"></span>
-                    <span th:if="${swaggerEnabled==true}" ><br/>Swagger uri: <a th:href="${swaggerURI}" target="_blank" th:text="${swaggerURI}"></a></span>
+                    Spoilers enabled: <span th:text="${spoilingEnabled}"></span><br/>
+                    Springdoc enabled: <span th:text="${springdockenabled}"></span>
```

Comment:
```suggestion
                    Springdoc enabled: <span th:text="${springdocenabled}"></span>
```

Understandable typo, probably search for this in more places